frontend/src/components/ModifyAccountForm.vue

```diff
@@ -52,11 +55,14 @@
                 </div>
                 <div class="form-group">
                   <label for="dniNie">DNI/NIE</label>
-                  <input type="text" v-model="user.dniNie" id="dniNie" name="dniNie" class="form-control" :class="{ 'is-invalid': submitted && $v.user.dniNie.$error }" />
+                  <input type="text" v-model="user.dniNie" id="dniNie" name="dniNie" class="form-control" :class="{ 'is-invalid': submitted && $v.user.dniNie.$error | error_dniNie}" />
```

Comment:
Aqui usas bit-wise OR ?  no es || ?

Reply:
this is weird, porque funciona con |...verdad?